src/metaArray.h

```diff
@@ -53,7 +53,7 @@ class METAIO_EXPORT MetaArray : public MetaForm
   // PUBLIC
 public:
   // Constructors & Destructor
-  MetaArray(void);
+  MetaArray();
```

Comment:
Incredible number of these!

Reply:
Yes.  There is an automated way to do this.  The recommendation for readability is to remove them.  I did this for ITK years ago.